src/cloud-api-adaptor/test/e2e/common_suite.go

```diff
@@ -572,3 +572,26 @@ func DoTestPodsMTLSCommunication(t *testing.T, e env.Environment, assert CloudAs
 	NewTestCase(t, e, "TestPodsMTLSCommunication", assert, "Pods communication with mTLS").WithPod(serverPod).WithExtraPods(extraPods).WithConfigMap(configMap).WithService(nginxSvc).WithSecret(serverSecret).WithExtraSecrets(extraSecrets).Run()
 
 }
+
+func DoTestKbsKeyRelease(t *testing.T, e env.Environment, assert CloudAssert) {
+
+	log.Info("Do test kbs key release")
+	pod := NewBusyboxCurlPod(E2eNamespace)
+	testCommands := []TestCommand{
+		{
+			Command:       []string{"curl", "http://127.0.0.1:8006/cdh/resource/reponame/workload_key/key.bin"},
```

Comment:
busybox ships wget if we want to use busybox

Reply:
The comment is reasonable to me, we don't need introduce new function `NewAlpineCurlPod` and can reuse `NewBusyboxPodWithName` directly. And change the command to:
```
[]string{"wget", "http://127.0.0.1:8006/cdh/resource/reponame/workload_key/key.bin"},
```

Reply:
As I tried, the command should be:
`wget -q -O- http://127.0.0.1:8006/cdh/resource/reponame/workload_key/key.bin`, we need parameters `-q -O-` here.

Reply:
So, command would be:
```
[]string{"wget", "-q", "-O-", "http://127.0.0.1:8006/cdh/resource/reponame/workload_key/key.bin"}
```